libcxx/docs/ReleaseNotes/22.rst

```diff
@@ -49,6 +49,7 @@ Implemented Papers
 - P2835R7: Expose ``std::atomic_ref``'s object address (`Github <https://llvm.org/PR118377>`__)
 - P2944R3: Comparisons for ``reference_wrapper`` (`Github <https://llvm.org/PR105424>`__)
 - P3168R2: Give ``std::optional`` Range Support (`Github <https://llvm.org/PR105430>`__)
+- P2442R1: Add ``std::views::chunk`` (`Github <https://llvm.org/PR171234>`__)
```

Comment:
We don't normally provide release notes for partially implemented papers (but the first entry does it anyway). In case somebody doesn't ask you to remove it I'd change it to:


```suggestion
- P2442R1: P2442R1: Windowing range adaptors: ``views::chunk`` and ``views::slide`` (`Github <https://llvm.org/PR171234>`__) (Implemented ``views::slide`` only)
```

I also created a sub-task for you and linked it: https://github.com/llvm/llvm-project/issues/171258 parent issue is: https://github.com/llvm/llvm-project/issues/105187


BTW. A previous stale attempts at implementing `slide` and `chunk`, which may be useful to you:

https://github.com/llvm/llvm-project/pull/85741
https://github.com/llvm/llvm-project/pull/67146